src/index.js

```diff
@@ -33,6 +33,9 @@ BetterLyrics.App = {
     try {
       if (document.readyState !== "loading") {
         BetterLyrics.App.modify();
+        document.addEventListener("blyrics-send-player-time", function(event) {
+          BetterLyrics.DOM.tickLyrics(event.detail.currentTime + 0.25);
```

Comment:
do we want the time offset here or moved to tickLyrics?

Reply:
i think moving it to `tickLyrics` might be a good idea. Checking the PR now

Reply:
I moved it, but I think we still want some sort of offset since there is still time spent scrolling the lyrics. (It is reduced from 0.75 to 0.25)